content/en/docs/collector/building/authenticator-extension.md

```diff
@@ -147,11 +141,8 @@ debug specific failures.
 
 ### Client authenticators
 
-A client authenticator is one that implements one or more of the following
-interfaces:
-
-- [go.opentelemetry.io/collector/config/configauth/GRPCClientAuthenticator](https://pkg.go.dev/go.opentelemetry.io/collector/config/configauth#GRPCClientAuthenticator)
-- [go.opentelemetry.io/collector/config/configauth/HTTPClientAuthenticator](https://pkg.go.dev/go.opentelemetry.io/collector/config/configauth#HTTPClientAuthenticator)
```

Comment:
These link targets no longer exists (the fragments are invalid), so I've reworked the prose and instead linked to the section headings introduced by https://github.com/open-telemetry/opentelemetry-collector/pull/12309/files (which hasn't been published yet, but the links are still valid -- the link fragments will be validated once 12309 is published.